proto/streamlit/proto/NumberInput.proto

```diff
@@ -0,0 +1,39 @@
+/**
+ * Copyright 2018-2019 Streamlit Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+*/
+
+syntax = "proto3";
+
+
+
+message NumberInput {
+  string id = 1;
+  string label = 2;
+  float default = 3;
+
+  oneof optional_min {
+    float min = 5;
+  }
+
+  oneof optional_max {
+    float max = 6;
+  }
+
+  float step = 7;
+
+  oneof optional_format {
```

Comment:
Remove `oneof`.

When `format` is not filled in Python, it will be `"" `in JS. And that's how we check whether the user passed in a format string.

Reply:
Done